src/lib/plugins/BrowserPlugin/BrowserPlugin.ts

```diff
@@ -13,6 +13,10 @@ const BASE_URL = 'baseUrl'
 
 export interface BrowserPluginProps {
   history: History
+  hostname?: string
+  host?: string
```

Comment:
Stupid question alert... Why host and hostname? Google seems to imply host is just hostname + port but we have port separately... is it sometimes the case that host =/= hostname + port?

Reply:
Hmmm. To be honest I just did it because the browser exposes them distinctly? if you look at window.location it has both fields.